src/js/SubmissionContainer.jsx

```diff
@@ -52,22 +53,26 @@ var SubmissionContainer = React.createClass({
       )
     }
 
-    if(code > 3) progress = <p>Progress component goes here</p>
+    if(code > 3) progress = <p className="third">Progress component goes here</p>
 
-    if(code > 6) editsContainer = <EditsContainer/>
+    if(code > 5) editsContainer = <EditsContainer/>
```

Comment:
Ideally, yes, but that implies we'll be streaming back the edit results rather than just the edit counts which I think we're avoiding for now.

Personally I think that getting as much data as soon as possible is a good thing, but it really depends on the speed of validation. If validation takes many minutes for large FIs, being able to investigate errors immediately I think is a usability win. But the extra streaming infrastructure might not be a good investment if validation takes < 20 seconds for 99% of users.

So for now I say we stick with only showing edits when validation has complete.